source/configure/plugins-configuration-settings.rst

```diff
@@ -291,9 +291,9 @@ Enable on specific channels
 .. include:: ../_static/badges/selfhosted-only.rst
   :start-after: :nosearch:
 
-**True**: Allow Channel Admins to enable or disable calls on specific channels. It also allows participants in DMs/GMs to enable or disable calls.
+**True**: Allow channel admins to enable or disable calls on specific channels. It also allows participants in DMs/GMs to enable or disable calls.
 
-**False**: Only System Admins will be able to enable or disable calls on specific channels.
+**False**: Only system admins will be able to enable or disable calls on specific channels.
```

Comment:
Not related to recording but as part of GA we removed this `Enable on specific channels` setting and renamed the `Enable on all channels` which is now `Test mode`, description is currently as follows:

>When test mode is enabled, only system admins are able to start calls in channels. This allows testing to confirm calls are working as expected.